src/cascade/model/integrands.py

```diff
@@ -7,20 +7,24 @@
 CODELOG, MATHLOG = getLoggers(__name__)
 
 
-def make_average_integrand_cases_from_gbd(execution_context, sexes):
+def make_average_integrand_cases_from_gbd(execution_context, sexes, include_birth_prevalence=False):
+    """
+    """
```

Comment:
Curious empty comment block?

Reply:
Well, crap. I started writing that and then I guess I got distracted.